### ISSUE TITLE
Question regarding the output

### ISSUE THREAD

**Issue by reshu-b7** [jakeret/tf_unet#103]
I am quite new to tensorflow and U-ne so I would highly appreciate if I can get some input on the following question. 

I am running the following code:

```
from tf_unet import unet, util, image_util
from pathlib import Path
import os

output_path = "/home/reshubisht/Pictures/t/test"
data_root = "/home/reshubisht/Pictures/t/train"
layers= 3
features_root = 16
epochs = 5
training_iters = 1

data_provider = image_util.ImageDataProvider(data_root + "/*.tif")

net = unet.Unet(channels= data_provider.channels, layers=layers, features_root=features_root,
                n_class=data_provider.n_class)
                
trainer = unet.Trainer(net, optimizer="adam")
path = trainer.train(data_provider, output_path, training_iters=training_iters, epochs=epochs)

x_test, y_test = data_provider(1)
prediction = net.predict(path, x_test)

print("Testing error rate: {:.2f}%".format(unet.error_rate(
        prediction, util.crop_to_shape(y_test, prediction.shape))))


img = util.combine_img_prediction(x_test, y_test, prediction)

util.save_image(img, "/home/reshubisht/Pictures/prediction.tif")
util.plot_prediction(x_test, y_test, prediction, save=True)

```

The input data contains several stacks of 3D images combined to one TIF image. The dimensions of the input data 30x512x512  and the dimension of the mask are the same. Below is the output from of the code in which only the segmented part of the 1 stack is available. The code gives no result for the remaining 2-30 stack. How can I get the results for all 30 stacks and save them in the end into one single "prediction.tif" file ?? Also is it possible to save only the prediction as an image ? I was searching through the documentation but I could not find a function which allows you to save just the predicted values. I do not want to combine the result. 

![prediction](https://user-images.githubusercontent.com/20737958/30511861-acc91e24-9aff-11e7-9484-bc81d04e3509.jpeg)

**Comment by LauraLaureus** [comment]
Have you got any information about this @reshu-b7 ? i have been the whole  afternoon studying the prediction code in order to use expresions like binaryImg = unet.predict(img) . The conclusion is that it uses softmax to predict which means values are real, between 0 and 1, and there is no posibility of segmentation because results are too little. 
i'm still studying this code...

**Comment by jakeret** [comment]
On your model you can use the predict [`function`](https://tf-unet.readthedocs.io/en/latest/tf_unet.html#tf_unet.unet.Unet.predict) then saving the image is up to you. I do that in the [toy example](https://github.com/jakeret/tf_unet/blob/master/demo/demo_toy_problem.ipynb)
